grammars/javascript.cson

```diff
@@ -1096,11 +1105,13 @@
     'patterns': [
       {
         'comment': 'functionCall(arg1, "arg2", [...])'
-        'begin': '\\b(\\w+)\\s*(\\()'
+        'begin': '(?:(\\d\\w*)|([a-zA-Z_$][\\w$]*))\\s*(\\()'
         'beginCaptures':
           '1':
-            'name': 'entity.name.function.js'
+            'name': 'invalid.illegal.js'
```

Comment:
Without `invalid.illegal` it would be `...(\\$+|(?:\\$+)?\\b[a-zA-Z_$][\\w$]*)...`